analysis/ukka/ckd_progression_viz.Rmd

```diff
@@ -343,27 +343,34 @@ d1 = rbind(
 ) %>%
   # select variables of interest
   select(Date, count, outcome)
+
+# Change date coding
+d1$Date_factor = factor(d1$Date)
+d1$Date_factor = recode_factor(d1$Date_factor, "2017-04-01" = "2016/17", "2018-04-01" = "2017/18", "2019-04-01" = "2018/19", "2020-04-01" = "2019/20", "2021-04-01" = "2020/21", "2022-04-01" = "2021/22") 
+
 ```
+
 #### Combination bar charts
-```{r, fig.width=7, fig.height=4}
+```{r, fig.width=9, fig.height=4}
 d2 = d1
 
-# Set plot limits
-plot_ymin = min(d2$count, na.rm=T)*0.9
-plot_ymax = max(d2$count, na.rm=T)*1.1
-
 # Set factor levels for outcome
 d2$outcome = factor(d2$outcome, levels = c("Outpatient appointments", "GP interactions", "ED presentations"))
 
 # Generate plot
-ggplot(d2, aes(x = Date, y = count, colour=outcome)) + 
+ggplot(d2, aes(x = Date_factor, y = count, fill=outcome)) + 
+    facet_grid(.~outcome) + xlab("") +
      geom_bar(stat="identity") + ylab("Count") +
```

Comment:
Since this is a bar chart, we need to shift from _colour=_ to _fill=_ as the colour will only edit the borders. The _facet_grid()_ is separating the outcomes into different panels